addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/utils/WifiSocketPacketConverter.java

```diff
@@ -268,7 +269,7 @@ private static String byteArrayToHexString(final byte[] array) {
     /**
      * Converts one partial byte array to one hexadecimal string.
      *
-     * @param array the byte array to convert.
+     * @param array  the byte array to convert.
```

Comment:
Did you setup a new IDE? Looks like something had happened to your formatter. I am fine with ignoring it. Wdyt?

Reply:
It has been an issue for a while: https://github.com/eclipse/smarthome/issues/6424

Reply:
My auto formatter works again as it should, after applying the [easy fix]( https://github.com/eclipse/smarthome/issues/6424#issuecomment-471728524).